airflow/providers/google/cloud/transfers/calendar_to_gcs.py

```diff
@@ -0,0 +1,178 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import json
+from datetime import datetime
+from tempfile import NamedTemporaryFile
+from typing import Any, List, Optional, Sequence, Union
+
+from airflow.models import BaseOperator
+from airflow.providers.google.cloud.hooks.gcs import GCSHook
+from airflow.providers.google.suite.hooks.calendar import GoogleCalendarHook
+
+
+class GoogleCalendarToGCSOperator(BaseOperator):
+    """
+    Writes Google Calendar data into Google Cloud Storage.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:TODO `howto/operator:GoogleCalendarToGCSOperator`
+
+    :param calendar_id: The Google Calendar ID to interact with.
+    :type calendar_id: str
+    :param destination_bucket: The destination Google cloud storage bucket where the
+        report should be written to. (templated)
+    :type destination_bucket: str
+    :param destination_path: The Google cloud storage URI array for the object created by the operator.
+        For example: ``path/to/my/files``.
```

Comment:
```suggestion
    :param destination_bucket: The destination Google Cloud Storage bucket where the
        report should be written to. (templated)
    :type destination_bucket: str
    :param destination_path: The Google Cloud Storage URI array for the object created by the operator.
        For example: ``path/to/my/files``.
```
Small nit for consistency.